src/redis-benchmark.c

```diff
@@ -412,6 +415,11 @@ static void resetClient(client c) {
     c->written = 0;
     c->pending = config.pipeline;
 }
+static long randomizeClientDurationJitter(long max_jitter_usec){
```

Comment:
I am thinking why not create a function to return the specific client ttl instead of just a random jitter calulation

Reply:
Done